codewof/programming/content/en/favourite-food/question.md

```diff
@@ -0,0 +1,6 @@
+# Favourite food
+
+Write a program that **prints** `What is your favourite food?`, then takes an input from the user,
+and then **prints** `Yummm... {x}` where `{x}` is the string the user typed in.
```

Comment:
This is like the opposite of NSON (and again, I missed it last time), there's a newline mid-sentence